turftopic/encoders/e5.py

```diff
@@ -26,7 +30,7 @@ class E5Encoder(SentenceTransformer, ExternalEncoder):
     """
 
     def __init__(
-        self, model_name: str, instruction: Optional[str] = None, **kwargs
+        self, model_name: str = "intfloat/multilingual-e5-small", instruction: Optional[str] = None, **kwargs
```

Comment:
I'm not sure I would do this. Unless the user makes an informed choice about which model they want, they shouldn't use E5 in my view. What do you think @jankounchained ?

Reply:
in my opinion, [multilingual-e5-small](https://huggingface.co/intfloat/multilingual-e5-small) is a sensible default.
almost anyone can run it on their machine, has a pretty good performance.

without defaults, that you can use to immediately check if the encoder works, what's the point of having the e5 encoder in the first place? the encoder provides nothing, and might even make the job harder, because the prefixing is hidden inside the class, out of reach of many users.
people might as well get the e5 embeddings on their own and then use them in turftopic.

that being said, we could go with no default & make a tutorial notebook instead. Or just give an example in the docstring.

@x-tabdeveloping it's your call.
I'm starting to like the docstring solution myself.